README.md

```diff
@@ -1,32 +1,30 @@
 # Prequisites
 1. Have Docker running
-2. Login to Jamsocket CLI `npx jamsocket@latest login`
+2. Have NodeJS >18 installed.
```

Comment:
does node need to be installed if the app is in python?

Reply:
This would just be to make sure that the Jamsocket CLI works.